src/views/NoResultsView.js

```diff
@@ -10,6 +10,8 @@ class NoResultsView extends Backbone.View {
         this.mediator.on('search:refinedSearch', this.onSearchInitiated, this);
         this.mediator.on('search:displayPreviousResults', this.onDisplayPreviousResults, this);
         this.mediator.on('app:home', this.onAppHome, this);
+
+        this.hideNoResultsMessage();
```

Comment:
I'm guessing this was added due to a failing (revived) test?

Reply:
Specifically, look in the `main_view.html` template, at the empty div for `no-results`.  (Not in this PR because that template was not changed).

Another alternative could be to just remove the test for "defaults to hidden" (and perhaps document that the parent container defines its initial state).